packages/app-store/googlecalendar/api/callback.ts

```diff
@@ -91,6 +93,10 @@ async function getHandler(req: NextApiRequest, res: NextApiResponse) {
       throw new HttpError({ message: "Internal Error", statusCode: 500 });
     }
 
+    // Update the user's profile photo with google profile photo if it's null
+    // Since we don't want to block the user from using the app if this fails, we don't await this
```

Comment:
This doesn't really work in serverless environments. If you don't await and the main endpoint responds first the call will be dropped.

Reply:
TIL. Can we just wrap it in a try/catch block and if it fails just continue on with the rest of the code?

Reply:
@zomars there's already a try/catch in the function it's self. I tested throw an error in the function and it still creates the credential. I'll add the await and one more change in a follow up PR.